### DISCUSSION TITLE
优化正版的更改皮肤

### DISCUSSION THREAD

**Discussion by DXJLSFB139** [Meloong-Git/PCL#712] [提问] [answered]
可以在点击头像时打开一个界面，里面有一个和HMCL一样的皮肤模型，可以预览修改后的皮肤，修改前的皮肤

当然我知道这个功能很累，看着做吧!只能说是个建议，当然这也会让PCL更优秀！

**Comment by WForst-Breeze** [comment, marked as answer]
感谢您的建议！但是想要让您的想法被采纳的话，请前往提交 [**新功能提案**](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&title=%E4%B8%BA%E4%BF%AE%E6%94%B9%E7%9A%AE%E8%82%A4%E6%B7%BB%E5%8A%A0%E9%A2%84%E8%A7%88&template=feature.yml) （标题我帮您换了一个更好一点的）
